packages/bsb-core/bsb/config/_attrs.py

```diff
@@ -986,24 +1016,104 @@ def tree(self, instance):
         return val
 
 
+class cfgreflist(cfglist):
+    """
+    cfglist that stores both the list of node references and their resolved values.
+    """
+
+    def __init__(self, *a, **k):
+        super().__init__(*a, **k)
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
```

convention, we only exceptionally used `*a, **k` because I was in a hurry when showing you the lambda :p